### ISSUE TITLE
short title, affiliation not rendered.

### ISSUE THREAD

**Issue by tcgriffith** [crsh/papaja#333]
**Describe the bug**

This seems to be caused by the latest upgrade from the bookdown tex template.From the same Rmd, the rendered tex files are different:

[old tex](https://github.com/tcgriffith/cosx_exps/blob/master/papaja_old.tex#L66-L100), short title and affiliations are in the tex

[new tex](https://github.com/tcgriffith/cosx_exps/blob/master/papaja_new.tex#L73-L80), short title and affiliations are missing

**To Reproduce**

Upgrade to bookdown 0.16
[here are the files for test2.Rmd generated test2.tex and rendered test2.pdf](https://github.com/tcgriffith/cosx_exps/tree/master/papaja_issue)

**Expected behaviour**

render short title/ affiliation

**Screenshots**

**Additional context**

Session info:

```
> xfun::session_info()
R version 3.6.1 (2019-07-05)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 18.04.3 LTS, RStudio 1.2.5019

Locale:
  LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8       
  LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
  LC_PAPER=en_US.UTF-8       LC_NAME=C                  LC_ADDRESS=C              
  LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

Package version:
  assertthat_0.2.1  backports_1.1.5   base64enc_0.1.3   BH_1.69.0.1       bookdown_0.16    
  broom_0.5.2       cli_1.1.0         compiler_3.6.1    crayon_1.3.4      digest_0.6.23    
  dplyr_0.8.3       ellipsis_0.3.0    evaluate_0.14     fansi_0.4.0       generics_0.0.2   
  glue_1.3.1        graphics_3.6.1    grDevices_3.6.1   grid_3.6.1        highr_0.8        
  htmltools_0.4.0   jsonlite_1.6      knitr_1.26        lattice_0.20.38   lifecycle_0.1.0  
  magrittr_1.5      markdown_1.1      methods_3.6.1     mime_0.7          nlme_3.1.142     
  packrat_0.5.0     papaja_0.1.0.9842 pillar_1.4.2      pkgconfig_2.0.3   plogr_0.2.0      
  plyr_1.8.4        purrr_0.3.3       R6_2.4.1          Rcpp_1.0.3        reshape2_1.4.3   
  rlang_0.4.2       rmarkdown_1.18    rstudioapi_0.10   stats_3.6.1       stringi_1.4.3    
  stringr_1.4.0     tibble_2.1.3      tidyr_1.0.0       tidyselect_0.2.5  tinytex_0.17     
  tools_3.6.1       utf8_1.1.4        utils_3.6.1       vctrs_0.2.0       xfun_0.11        
  yaml_2.2.0        zeallot_0.1.0    
```

**Comment by ericpgreen** [comment]
Came here to say the same thing. R 3.6.0 (2019-04-26), Ubuntu 18.04.2 LTS, papaja_0.1.0.9842

**Comment by tcgriffith** [comment]
I tried other bookdown versions (0.10~0.16) and still had the same problem.

**Comment by ericpgreen** [comment]
I cannot reproduce the error on my local machine (Mac), but it does appear on my Ubuntu machine as I noted yesterday.

**Comment by tcgriffith** [comment]
I'll dig deeper while I got some time though.

**Comment by crsh** [comment]
That would be very helpful, thank you. My hunch is, that this could be related to your `rmarkdown` version.

**Comment by tcgriffith** [comment]
Yes, it's `rmarkdown`

Tested with the previous version(1.17), running head and affiliation are fine.

**Comment by crsh** [comment]
Thanks, I will look into it.

**Comment by crsh** [comment]
I think, this is now fixed on the development branch (`devtools::install_github("crsh/papaja@devel")`). Give it a try and let me know if it works for you.

**Comment by tcgriffith** [comment]
It seems that the devel version of papaja used your rmdfiltr package and caused some error:

```
output file: rna_coup_ver2.knit.md

Error in rmdfiltr::add_wordcount_filter(args, error = FALSE) : 
  unused argument (error = FALSE)
Calls: <Anonymous> -> <Anonymous> -> pdf_pre_processor
Execution halted
```

**Comment by crsh** [comment]
Ah, yes. You need to update to the latest version.

Von meinem Handy gesendet

**Comment by tcgriffith** [comment]
Hi, crsh,

In your dev branch, the `add_wordcount_filter` parameter name is `error`.
https://github.com/crsh/papaja/blob/devel/R/apa6_formats.R#L391

but in your rmdfiltr package, the parameter name is `report`.
https://github.com/crsh/rmdfiltr/blob/master/R/add_lua_filter.R#L51

Did you forget to push your local rmdfiltr to github?

**Comment by crsh** [comment]
Hi TC,

sorry for the short incomprehensible reply. I was on the go all day yesterday. I have been in the process iteratively updating `rmdfiltr` in response to issues raised by the CRAN team. The version that fixes the problem you mention (0.1.2) has been published on CRAN, but you are right, I forgot to push those commits to GitHub after it was accepted (it's on my machine at home, so I'll have to do it later).

In short, reinstalling from CRAN will give you the most current version and should fix the problem you mention above. ;)

**Comment by tcgriffith** [comment]
I assumed that github version was more updated than cran. haha
